spyder/app/mainwindow.py

```diff
@@ -1465,12 +1468,13 @@ def resizeEvent(self, event):
 
     def moveEvent(self, event):
         """Reimplement Qt method"""
-        if not self.isMaximized() and not self.layouts.get_fullscreen_flag():
-            self.window_position = self.pos()
-        QMainWindow.moveEvent(self, event)
+        if hasattr(self, 'layouts'):

```

Comment:
I think we decided `layouts` is a core plugin, so it can't be disabled. In that case, is this check necessary?

Reply:
This is necessary in order to prevent some spurious errors in some of the tests